### PR TITLE
feat: rm ecommerce

### PR DESCRIPTION
**Pull request by Abdur-rahmaanJ**

### CODE REVIEW

**Review by rehmanis — CHANGES_REQUESTED** [review]
After running 
```bash
python mange.py initialize
python manage.py runserver
```
App crashes when http://localhost:5000/ is loaded. Crashes at:
```bash
  File "\shopyo\shopyo\modules\www\view.py", line 52, in index
    return render_template(get_setting('ACTIVE_FRONT_THEME')+'/index.html')
jinja2.exceptions.TemplateNotFound: ecommerceus/index.html
```

**Comment by Abdur-rahmaanJ** [comment]
@rehmanis sure not a cache issue on your side? You need to remove your config.py and config.json like start on a really clean slate. The tests would have failed otherwise

**Comment by rehmanis** [comment]
> @rehmanis sure not a cache issue on your side? You need to remove your config.py and config.json like start on a really clean slate. The tests would have failed otherwise

Yes test are failing. Let me try again.

**Comment by rehmanis** [comment]
@Abdur-rahmaanJ yeah working fine now. Was the `config.json` file.

**Review by rehmanis — APPROVED** [review]
